chasm/lib/scheduler/invoker_tasks.go

```diff
@@ -121,15 +127,15 @@ func (e *InvokerExecuteTaskExecutor) Execute(
 		invokerRef,
 		func(i *Invoker, ctx chasm.Context, _ any) (struct{}, error) {
```

Comment:
nit: re. the `struct{}`, the intention of that is for the method to return something, i.e. the scheduler created here. and this method doesn't have to capture the variable defined on L115.